docs/src/references.md

```diff
@@ -0,0 +1,4 @@
+# References
```

Comment:
I'm unable to leave a comment in the right location, so I'll just dump it here: In index.md, would be good to spell out HMC as Hamiltonian Monte Carlo the first time it's mentioned.